src/target/riscv/debug_reg_printer.h

```diff
@@ -2,6 +2,12 @@
 
 #include "debug_defines.h"
 
+enum riscv_debug_reg_show {
```

Comment:
Please, could you give a descriptive name to this enum, e.g. `display_mode` (or similar), rather than `show`.

```suggestion
enum riscv_debug_reg_display_mode {
```

Reply:
Correction per [kr-sc's comment](https://github.com/riscv/riscv-openocd/pull/973#issuecomment-1849825669): This change should go to the debug spec repository instead (riscv-debug-spec).